.github/workflows/action-lint.yml

```diff
@@ -0,0 +1,10 @@
+name: Lint GitHub Actions
+on:
+  workflow_call:
+
+jobs:
+  action-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: reviewdog/action-actionlint@v1
```

Comment:
I propose we set `fail_on_error: true` to prevent lint errors.

Of course, we may need to run around and fix a few things once we introduce it.

Reply:
Good call, added